python/lsst/cp/verify/verifyCalib.py

```diff
@@ -86,15 +96,33 @@ class CpVerifyCalibTask(pipeBase.PipelineTask, pipeBase.CmdLineTask):
     ConfigClass = CpVerifyCalibConfig
     _DefaultName = 'cpVerifyCalib'
 
-    def run(self, inputCalib):
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
```

Comment:
This `runQuantum` doesn't do anything that the default wouldn't, so it doesn't need to be here.